reflectable/lib/reflectable.dart

```diff
@@ -4,8 +4,6 @@
 
 // This is the main library in package reflectable.
```

Comment:
This comment doesn't provide much. Perhaps we should delete it?

Reply:
Right, done.